src/app/models/sap_model.php

```diff
@@ -143,6 +143,7 @@ public function get_tasks_with_submissions($user_id, $mission_id, $delete_reject
             $mission_id
         );
         // TODO: Refactor this to make it faster than O(n^2)
+        // Get latest submission for each task
```

Comment:
So, since the submissions are in ascending order by id, at the end of this loop, the latest submission will be the one that gets put into the `$task`, right?